docs/fast_data/configuration/projection_storer.md

```diff
@@ -0,0 +1,1048 @@
+specified in the connection string, although it would be recommended to set it in case the connection string is shared
+among multiple 
+
+This is an example of storage configuration when `mongodb` is selected as type:
+
+```json
+"storage": {
+  "type": "mongodb",
+  "configuration": {
+    "url": "mongodb://localhost:27017/fast-data-inventory-local",
+    "database": "fast-data-inventory-local"
+  }
+}
+```
+
+### Projections Config
+
+This section of the configuration provides all the details related to each projection associated to an instance of the
+Projection Storer service. The content of this property is mapping between projection names and their configuration the input and output specification together with the mapping configuration that
+instructs the service on how to transform each ingestion event into a projection record and where to store it.
+
+#### Topics
+
+| Property    | Type     | Required | Default |
+|-------------|----------|----------|---------|
+| `ingestion` | `object` | &check;  |         |
+| `prUpdate`  | `object` | &check;  |         |
+
+In this section are specified for each projection their input channel (_ingestion_), from which change events on the source
+system (System Of Records) will be read, and the output channel (_prUpdate_), where update notifications will be emitted
+to trigger Fast Data downstream components.
+
+##### Ingestion
+
+```json
+{
+  "ingestion": {
+    "name": "galaxy.fast-data.DEV.orders.ingestion"
+  }
+}
+```
+
+##### Projection Record Update
+
+```json
+{
+  "prUpdate": {
+    "name": "galaxy.fast-data.DEV.customers.pr-update"
+  }
+}
+```
+
+#### Custom Storage Namespace
+
+| Property                 | Type     | Required | Default             |
+|--------------------------|----------|----------|---------------------|
+| `customStorageNamespace` | `string` | -        | `<projection-name>` |
+
+Represents the name employed on the storage system for knowing where to store the records of this projection. By default,
+this value corresponds to the name of the projection itself.
+
+For example, when using MongoDB as storage system, this field allows to define a custom name for the collection where the
+records of this projection will be saved.
+
+#### Primary Keys
+
+| Property      | Type       | Required | Default |
+|---------------|------------|----------|---------|
+| `primaryKeys` | `string[]` | &check;  |         |
+
+It is the list of fields names that identifies a record for this projection. The names contained in this list are relative
+to the fields of the source record contained in the ingestion event. There should always be at least one name in
+this list, so that it is possible to uniquely connect records of different projections among them.
+
+:::caution
+Primary keys array **must** contain the names of the fields that are found on the ingestion event document. In case
+the `targetField` property is employed to rename a field belonging to the projection primary key, before
```

Comment:
what's the `targetField`? if possible add a link

Reply:
Honestly I didn't really get what do you mean here. If I rename the primary keys should I put in this list the updated name or not?

Reply:
It should be clearer. I moved the infobox after the introduction `targetField` and clarified the description.